release-notes/release-notes-v0.5.0.md

```diff
@@ -0,0 +1,433 @@
+49871df0c35 Require CJDNS starts with 0xFC (#2356)
+3e7fade2240 Add latest ProtocolVersion (#2332)
+13443fd0d1e Implement BIP 155 addrv2 messages (#2321)
+b3d70f559a6 Fix TLV parsing for non-standard strings (#2312)
+0c03aa9c24d Add function to validate OracleAnnouncementV0TLV's signature (#2308)
+28c37ccf1fe trivial: Add a sha256 hash field on all TLV (#2300)
+b638c6837e6 Move english word list to be represented by a Vector in memory (#2287)
+18dfbed8c90 Add helper OracleEventTLVV0.maturation method (#2267)
+e4194220c1e Update Oracle TLVs (#2185)
+7178cb7136f Fixed P2SH(Segwit) bug and brought down DLC-used PSBT functionality (#2140)
+ddf39057fe0 Only Validate PSBT BIP143 vulnerability on signing (#2204)
+02c10fd89ed Pretty Fee Rate toStrings (#2210)
+c275a8c8aa6 Make tx bytes functions lazy vals (#2180)
+c5be81d5e21 Add address descriptors (#2176)
+2e1b4d04919 P2SHTxSigComponent constructor now detects witness data (#2169)
+7bbc89fb33f Optimize GetHeadersMessage.fromBytes (#2131)
+4ac93660d2c Add BIP45 Multisig Purpose (#2103)
+f69678d74bf Allow any HDCoinType (#2097)
+fa31609daca Enable AddrMessageTest (#2106)
+d158f4fcb08 Calculate HRP from network param instead of the inverse (#2079)
+c2315082d8c Create ExtPrivateKeyHardened (#2073)
+856f88f0ddb Initial SigNet support (#2057)
+
+### Crypto
+
+ECDSA Adaptor signatures have been added in a basic Java implementation. These should be used with caution as they are
+experimental and still being developed.
+
+Low R Signing should now use the same algorithm as Bitcoin Core.
+Previously our first attempted signature would use different added entropy.
+
+#### Commits
+
+70d014f7b26 Fixed Low R signing (#2408)
+c7e5c634f3a Brought down ecdsa adaptor signatures implemented in scala from the dlc-crypto branch (#2034)
+5bcf3e2a533 Introduced NFC normalization for strings in CryptoUtil and added String hashing functions (#2102)
+
+### Db Commons
+
+Various improvements to initializing and parsing an `AppConfig`.
+
+#### Commits
+
+6b2237d2e81 Fix overrides AppConfig.configOverrides with DLCOracleAppConfig, also remove hardcoded value out of
+AppConfig.configOverrides so people have to implement it (#2209)
+ca5324c6a22 2020 10 12 issue 2164 (#2173)
+1997f22e7d3 Refactor db configuration to use the key 'user' rather than 'username' inline with slick documentation (
+
+# 2184)
+
+56b385a0e79 Create fromConfig functions for AppConfigs (#2170)
+0e41c29a3fe 2020 10 08 issue 2147 (#2153)
+da2f3232cd0 2020 10 06 explicit classloader (#2148)
+475ed19df66 2020 10 05 redo config (#2121)
+4aec33f38cc Move configuration for sqlite into the 'bitcoin-s' key (#2113)
+
+### DLC Oracle
+
+The DLC Oracle is a new module in Bitcoin-S!
+The DLC Oracle allows you to attest to events and sign their outcomes for DLC participants.
+
+This is still in beta and is likely to have some breaking changes in the future. As of this release, it is up to date
+with the [DLC Spec](https://github.com/discreetlogcontracts/dlcspecs).
```

Comment:
Might want to mention this can be used as a stand alone binary rather than a library and is included below?